lib/membrane_mp4/muxer/isom.ex

```diff
@@ -21,7 +21,7 @@ defmodule Membrane.MP4.Muxer.ISOM do
     caps: Membrane.MP4.Payload,
     availability: :on_request
 
-  def_output_pad :output, caps: :buffers
+  def_output_pad :output, caps: {RemoteStream, type: :bytestream, content_format: MP4}
```

Comment:
Not sure whether this should be `:bytestream`. I feel like it should be `:packetized` as we are probably sending well defined chunks in each Membrane.Buffer :thinking: